app/general/tests/test_document_admin.py

```diff
@@ -8,6 +8,8 @@
 
 
 class TestDocumentForm(unittest.TestCase):
+    # The methodName parameter must be camelCase as it overrides the unittest.TestCase equivalent
+    # noinspection PyPep8Naming
```

Comment:
Maybe check why ruff didn't complain. This rule feels like it should have triggered:
https://docs.astral.sh/ruff/rules/invalid-argument-name/
(unless it knows not to complain in unittest - a few standard library modules use camelcase from before pep-8 was widely followed)

Apart from the two linters, this introduces a second syntax for linter override (we currently use "noqa") and check names, so maintainability isn't ideal this way. Since we can't automate PyCharm's, we're not even going to consider it.

Feel free to provide notes on Pycharm settings under doc/.

Reply:
> Maybe check why ruff didn't complain. This rule feels like it should have triggered:

This could be related to what we discussed on Slack yesterday - we had very few ruff lints enabled it turned out :sweat: I don't think that we had R (pep8 naming) enabled

> Apart from the two linters, this introduces a second syntax for linter override (we currently use "noqa") and check names, so maintainability isn't ideal this way. Since we can't automate PyCharm's, we're not even going to consider it.

Agreed it's better to stick to one - hopefully PyCharm also will respect `noqa`

Reply:
Perhaps it's best to drop this commit and then revisit it in the PR dedicated to adding (and fixing) more Ruff lints?